### ISSUE TITLE
not all tabs in dialog appear in edge , thought they appear in chrome 

### ISSUE THREAD

**Issue by eassa** [angular/material#9770]
some dialogs work others dont i haveno idea why , 
1. i can change tabs by click on one ,then moving right arrow with the keyboard and it will change (but still wont show me the tab)
2.the disabled tab is hidden as well,  even if i enable it (through checkbox) it stays hidden  

thats my code: 

```
<md-dialog aria-label="Block Settings"  style="overflow:hidden">
	<form ng-cloak novalidate name="multiline">
		<md-toolbar>
			<div class="md-toolbar-tools">
				<h2><md-icon class="material-icons">settings</md-icon> {{title}}</h2>
				<span flex></span>
				<md-button class="md-icon-button" ng-click="cancel()">
					<md-icon class="material-icons">close</md-icon>
				</md-button>
			</div>
		</md-toolbar>
		<md-dialog-content  ng-keypress="($event.which === 13)?save():0" md-dynamic-height style="max-width:800px;max-height:810px;">	
			<div id="multiline" style="overflow:hidden">
				<md-tabs md-selected="selectedTab" md-no-pagination md-dynamic-height md-align-tabs="top">	   
					<md-tab ng-click="" label="Anchors">
						<ng-include src="'anchors.html'"></ng-include>
					</md-tab>
					<md-tab ng-click="" label="Settings" >
						<ng-include src="'anchorsSettings.html'"></ng-include>	
					</md-tab>		
				</md-tabs>
				
			</div>
		</md-dialog-content>		
		<md-dialog-actions layout="row">
			<md-button ng-click="cancel()" class="md-raised" aria-label="Cancel">Cancel</md-button>
			<span flex></span>
			<md-button ng-click="save()" class="md-raised md-primary" aria-label="Save"><md-icon class="material-icons modalbtn">&#xE161;</md-icon> Ok</md-button>
		</md-dialog-actions>
	</form>
</md-dialog>
```

**Comment by twizzzlers** [comment]
My tabs are also not appearing in chromium, in firefox they work perfectly...

**Comment by eassa** [comment]
i am not having any tab problem with chrome .
I am using version 1.1.1 , and last chrome version . (Try to update )

**Comment by twizzzlers** [comment]
Other person gave me a hacky way of fixing my problem, try to see if works for you while they dont resolve this:

`md-pagination-wrapper {
  width:auto !important;
}` 

#9429

**Comment by eassa** [comment]
Tomorrow at work i will check it and update you :) thx :)

**Comment by eassa** [comment]
yup, it works like a charm :) thank you

**Comment by bradrich** [comment]
This issue also occurs when using `md-tabs` within a `md-card` as well.

**Comment by manish2535** [comment]
md-pagination-wrapper is not getting the right width hence wrapping the last tab making it invisible.

**Comment by clshortfuse** [comment]
From the sound of the this, this might be fixed with #9451 which is built to solve various issues related to tabs inside dialogs.

The PR already has 3 issues assigned to it. I'm closing this is a duplicate of one of those 3, namely: https://github.com/angular/material/issues/9779

@eassa, if I'm mistaken and this issue isn't fixed with the PR, please include a working demo codepen to better show the issue.
